custom/ewsghana/handlers/__init__.py

```diff
@@ -12,3 +12,5 @@
 
 MS_RESOLVED_STOCKOUTS = _('The following commodities %(products_names)s are now available at the %(ms_type)s. '
                           'Please place your order now.')
+
+STOP_MESSAGE = _('You have requested to stop reminders to this number.')
```

Comment:
I found the original message, it's here: https://github.com/dimagi/rapidsms-logistics/blob/2ef22c430ebab7ecaabf9ca6e60a00d3bef1f29a/logistics/handlers/stop.py#L20